Retail/deploy.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+bucket=$1
+echo "Bucket is $bucket"
+
+pwd
+
+mkdir poc_data
+mkdir poc_data/data
+mkdir poc_data/metadata
+
+echo "Local copy sync Retail"
+
+wget -P poc_data/data https://code.retaildemostore.retail.aws.dev/csvs/interactions.csv
+wget -P poc_data/data https://code.retaildemostore.retail.aws.dev/csvs/items.csv
+wget -P poc_data/data https://code.retaildemostore.retail.aws.dev/csvs/users.csv
+
+sleep 60
+echo "Starting the copy to S3 data"
+
+aws s3 cp ./poc_data/data/users.csv s3://$bucket/train/retail/users.csv
+aws s3 cp ./poc_data/data/interactions.csv s3://$bucket/train/retail/interactions.csv
+aws s3 cp ./poc_data/data/items.csv s3://$bucket/train/retail/items.csv
+
+sleep 60
```

Comment:
What is this needed? Remove?

Reply:
This has been in here for a while I can test if it can be removed when I add the news immersion day which I have currently separated out from News and Retail one.

Reply:
I think the sleep was in there for the old media flow that ran the notebook code to prepare the grouplens datasets. For retail, we're just copying files into place so once the copies are done, it's good to go.